web/src/css/flowdetail.less

```diff
@@ -21,6 +21,21 @@
         }
     }
 
+    .close-button {
+        margin-top: 2px;
+        padding-left: 10px;
+        padding-right: 7px;
+        color: grey;
+        font-weight: 500;
+        font-size: 15px;
+        cursor: pointer;
+        border: none;
+    }
```

Comment:
There are still some leftovers here. For example, `cursor: pointer;` is unnecessary for a button.

Reply:
right, just `cursor` and `font-weight`, the others are necessary I think

Reply:
now?